test/hw4/test_task2.py

```diff
@@ -0,0 +1,38 @@
+from unittest.mock import patch
+
+import pytest
+
+from All_home_works.hw4.task2 import count_dots_on_i
+
+URL = "https://example.com/"
```

Comment:
Please inline it, and put something that does not look like real URL, to be sure that it will not be called.